spec/index.html

```diff
@@ -248,5 +249,155 @@ <h2>Basic Concepts</h2>
+              constraints or conventions that apply to a YAML-LD document according to [[RFC6906]].
+              A profile does not change the semantics of the resource representation
+              when processed without profile knowledge, so that clients both with
+              and without knowledge of a profiled resource can safely use the same
+              representation. The <code>profile</code> parameter MAY be used by
+              clients to express their preferences in the content negotiation process.
+              If the profile parameter is given, a server SHOULD return a document that
+              honors the profiles in the list which it recognizes,
+              and MUST ignore the profiles in the list which it does not recognize.
+              It is RECOMMENDED that profile URIs are dereferenceable and provide
+              useful documentation at that URI. For more information and background
+              please refer to [[RFC6906]].</p>
+            <p>This specification defines seven values for the <code>profile</code> parameter.</p>
+            <dl>
+              <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-expanded-document-form">expanded YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-compacted-document-form">compacted YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#context</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11#dfn-context-document">YAML-LD context document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-flattened-document-form">flattened YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#frame</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11-framing#dfn-frame">YAML-LD frame document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-framed-document-form">framed YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#extended</code></dt>
+              <dd>To request or specify <a data-lt="extended document form">extended YAML-LD document form</a>.
+                <div class="ednote">
+                  This is a placeholder for specifying something like an
+                  <dfn data-lt="extended-document-form">extended YAML-LD document form</dfn>
+                  making use of YAML-specific features.
+                </div></dd>
+            </dl>
+            <p>All other URIs starting with <code>http://www.w3.org/ns/json-ld</code>
+              are reserved for future use by JSON-LD specifications.</p>
```

Comment:
Do we need a  http://www.w3.org/ns/yaml-ld ?
Does URI reservation belong to media type registration?
If the answers are no, then remove.

```suggestion
```

Reply:
We can certainly remove the paragraph, but the overriding specification is json-ld and yaml-ld uses this namespace. I don't think there's a hard-and-fast rule against creating additional namespaces, but as the spec is so aligned with JSON-LD I think it's probably best to keep such IRIs localized, at least for now.

Reply:
@gkellogg OK to use http://www.w3.org/ns/json-ld for yaml-ld if needed.